lib/geojson2image.rb

```diff
@@ -24,11 +24,41 @@ def initialize(json: nil, width: nil, height: nil, padding: nil,
         @height_padding = 0
         @global_ratio = 0
         @png = ChunkyPNG::Image.new(@width, @height, @background)
+        set_configs_and_draw
       rescue Oj::ParseError
         puts "GeoJSON parse error"
       end
     end
 
+    def to_image
+      @png.save(@output)
+    end
+
+    def to_data_url
+      @png.to_data_url
+    end
+
+    private
```

Comment:
Several methods are now going to change their visibility scope, which includes `get_points', `get_boundary`, `transform_point`, and `draw`. Is that intentional?

Reply:
> Yes, because these methods modify instance variables and there are not any getters.
> Calling this method outside the scope of this class doesn't make any sense.
> So, I believe only `to_image` was meant to be the instance method in the first place.